gulpfile.js

```diff
@@ -1,210 +1,182 @@
-var gulp = require('gulp-4.0.build');
-var path = require("path");
-var webpack_stream = require('webpack-stream');
-var ON_DEATH = require('death')({ debug: false });
-
-var watch = require("gulp-watch");
-var del = require('del');
-var sass = require('gulp-sass');
-var openfinLauncher = require('openfin-launcher');
-var configPath = path.join(__dirname, '/configs/finConfig.json');
-//new
-var StartupConfig = require("./configs/other/server-environment-startup");
-const webpack = require('webpack');
-var chalk = require('chalk');
+// #region Imports
+// NPM
+const chalk = require("chalk");
+const { exec, spawn } = require("child_process");
+const ON_DEATH = require("death")({ debug: false });
+const del = require("del");
+const gulp = require("gulp-4.0.build");
+const sass = require("gulp-sass");
+const watch = require("gulp-watch");
+const merge = require("merge-stream");
+const launcher = require("openfin-launcher");
+const path = require("path");
+const webpack = require("webpack-stream");
+
+// local
+const webpackFilesConfig = require("./build/webpack/webpack.files.js")
+const webpackServicesConfig = require("./build/webpack/webpack.services.js")
+// #endregion
+
+// #region Constants
```

Comment:
what's region do? or is that just for you?

Reply:
It is organizational. In VS code, it folds the region.